govuk/login/template.ftl

```diff
@@ -107,11 +107,9 @@
     <header role="banner" id="global-header" class="${properties.kcHeaderClass!} with-proposition">
       <div class="header-wrapper">
         <div class="header-global">
-          <div class="header-logo">
-            <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" id="logo" class="content">
-              <img src="${url.resourcesPath}/vendor/govuk_template/images/gov.uk_logotype_crown_invert_trans.png?0.22.1" width="36" height="32" alt=""> GOV.UK
+            <a href="/" class="govuk-header__tna-logo">
+              <img src="${url.resourcesPath}/vendor/tna_resources/tna-horizontal-white-logo.svg?0.22.1" width="220px" alt=""> 
```

Comment:
Like I said in the [frontend PR](https://github.com/nationalarchives/tdr-transfer-frontend/pull/101/commits/70baf2a4969082e8f8d6f258c5954740c5a25330), the 220px width was a temporary change to make the logo look OK for the demo! Now that we've talked to Digital Services about how to use the horizontal logo, we should [style it so that it's vertically centred without hacking the width](https://national-archives.atlassian.net/browse/TDR-242).